### PR TITLE
Linux open cl

### PR DESCRIPTION
**Pull request by jeremad**
## Summary
Fix CUDA + OpenCL build on linux

### CODE REVIEW

**Comment by klem23** [comment]
Cuda 8 is lacking opencl support ?
Except the CI on cuda 8 everything seems fine

**Comment by jeremad** [comment]
There is a conflicting library between cuda and opencl, I have to track it down

**Comment by jeremad** [comment]
@w-m @klem23 : updated :)

File: docker/base.dockerfile [hunk 1]
```diff
@@ -22,13 +23,16 @@ RUN apt update && apt install -y \
   librtmp-dev \
   libx264-dev \
   ninja-build \
+  ocl-icd-opencl-dev \
```
Comment (w-m) [inline docker/base.dockerfile]: Should we put all of those in doc/BUILD.md as well?

Comment (jeremad) [reply docker/base.dockerfile]: :+1:

**Pull request merged**